### ISSUE TITLE
How long can I get the pwd of the dataset after submit the form?

### ISSUE THREAD

**Issue by asurada404** [kv2000/deepFashion3D#6]
Hi, thanks for you to share the dataset.
I wonder to know how long I can get the pwd of the dataset after submit the form?

**Comment by xiecookie** [comment]
Hello, Could I ask whether you get the pwd?I filled the form too and also don't get reply.

**Comment by asurada404** [comment]
Yes, You can send an email to the corresponding author to ask pwd directly.
